AAI/AAIConnectProfile.md

```diff
@@ -144,43 +138,80 @@ specific assignment within the organization that made the assertion.
     organization that has the authority to make the assertion on behalf of the
     user and is responsible for making and maintaining the assertion.
 
+<a name="term-visa-issuer"></a>
+**Visa Issuer** -- A service that signs [Visas](#term-visa). This service:
+* may be a [Broker](#term-broker) itself
+* may be used by a [Broker](#term-broker) as part of collecting [Visas](#term-visa).
+
 {% hr2 %}
 
 ## Relevant Specifications
 
-[OIDC-Core](http://openid.net/specs/openid-connect-core-1_0.html) -
-        Authorization Code Flow will generate id_tokens and
-        access_tokens from the Broker.
+<a name="ref-iana-jwt"></a>
+[[IANA-JWT]](https://www.iana.org/assignments/jwt/jwt.xhtml) -- Standard JWT [Claim](#term-claim) name assignments, 
+Internet Assigned Numbers Authority
 
-[RFC7519](https://tools.ietf.org/html/rfc7519) - JSON Web Token (JWT) - Specific implementations MAY extend
-        this structure with their own service-specific JWT claim names as top-level
-        members of this JSON object. The JWTs specified here follow JWS
-        specification from [RFC7515 JWS](https://tools.ietf.org/html/rfc7515).
+<a name="ref-oidc-core"></a>
+[[OIDC-Core]](http://openid.net/specs/openid-connect-core-1_0.html) -- OpenID Connect Core 1.0 (OIDC) -- 
+        [Authorization Code Flow](http://openid.net/specs/openid-connect-core-1_0.html#rfc.section.3.1) 
+        will generate id_tokens and access_tokens from the [Broker](#term-broker).
 
-[RFC7515](https://tools.ietf.org/html/rfc7515) - JSON Web Signature (JWS) is the
-        specific JWT to use for this spec.
+<a name="ref-oidc-client"></a>
+[[OIDC-Client]](https://openid.net/specs/openid-connect-basic-1_0.html) -- OpenID Connect Basic Client Implementer's Guide 1.0 
 
-[RFC5246](https://tools.ietf.org/html/rfc5246) - Transport Layer Security.
-        Information passed among clients, Applications, Brokers, and Passport
-        Clearinghouses MUST be protected using TLS.
+<a name="ref-oidc-discovery"></a>
+[[OIDC-Discovery]](https://openid.net/specs/openid-connect-discovery-1_0.html) -- OpenID Connect Discovery 1.0
 
-[OIDC-Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html)
+<a name="ref-passport"></a>
+[[Passport]](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md) - GA4GH Passport Specification, Data Use and Researcher Identity (DURI) Work Stream
```

Comment:
The referenced Passport spec does not define what Passport and Visas are, just their formats.